src/index.js

```diff
@@ -233,5 +255,6 @@ function getClassName(parseClass) {
 
 module.exports = {
   ParseServer: ParseServer,
-  S3Adapter: S3Adapter
+  S3Adapter: S3Adapter,
```

Comment:
not a big fan of exposing those adapters that way.
Maybe we could do `lib/Adapters/index.js` that would export all available adapters.
then 

const AvailableAdapters = require('parse-server/lib/Adapters');